digitalbitbox/bitbox-wallet-app
# onramp: moonpay MVP

No address or any other kind of secure verification.
We're at the mercy of Moonpay and relying on a non-compromized app
which is kind of against our threat model.

To test this, use:

- working fake CC: 4000 0231 0466 2535, 12/2020, 123
- declined CC: 4008 3708 9666 2369, 12/2020, 123
- TBTC and TETH work; send them back to `tb1q45h8zexwztmz3nyd8gmkxhpavdsva4znwwhzvs` and `0xc216eD2D6c295579718dbd4a797845CdA70B3C36` respectively
- KYC will always succeed; just click on "submit anyway"
- you'll have to provide a valid email addess because they send an check code; I sometimes use https://fakermail.com/ for this
